### DISCUSSION TITLE
Is there anyway to create a workflow that is triggered when a record in a postgres database reaches a certain value / new record inserted ...etc

### DISCUSSION THREAD

**Discussion by HadiDghaily** [elsa-workflows/elsa-core#3798] [Q&A]
i would like to create workflows that are triggered by CRUD operations in the database or when a record in a table reaches a certain value.
is this feasible in elsa? is there a package i could use for that... etc

any help would be highly appreciated as i cant find any documentation for the elsa packages at all.

**Comment by sfmskywalker** [comment]
There's nothing for this yet in Elsa, but it's something that I'll be needing for a customer in the not too distant future. I'll take this discussion as a reminder to create a roadmap item for it 😄
